framework/aster-frame/src/vm/space.rs

```diff
@@ -227,6 +227,7 @@ impl VmIo for VmSpace {
             }
```

Comment:
Just a reminder when rebasing. The entire `aster_frame::vm` module is renamed to `aster_frame::mm`.